x-pack/plugins/secops/public/containers/helpers.ts

```diff
@@ -12,4 +12,4 @@ import { ESQuery } from '../../common/typed_json';
 export const createFilter = (filterQuery: ESQuery | string | undefined) =>
   isString(filterQuery) ? filterQuery : JSON.stringify(filterQuery);
 
-export const getDefaultFetchPolicy = (): FetchPolicy => 'cache-and-network';
+export const getDefaultFetchPolicy = (): FetchPolicy => 'network-only';
```

Comment:
Adding a comment in case this was accidental, but I think this is intentional for KQL and other issues.

Reply:
Since this is intentional, I would put a comment or two about the issues we are seeing or write up the issues and put the issue linked to it so we know why we are using `network-only`